F1/F1/Race/RaceScheduleTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  RaceScheduleTableViewCell.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/20.
+//
+
+import UIKit
+
+class RaceScheduleTableViewCell: UITableViewCell {
+
+    // MARK: IBOutlet
+    @IBOutlet weak private var raceDay: UILabel!
+    @IBOutlet weak private var raceMonth: UILabel!
+    @IBOutlet weak private var session: UILabel!
+    @IBOutlet weak private var time: UILabel!
+
+    // MARK: Functions
+    func populateWith(sessionName: String, sessionDate: DateComponents, sessionTime: String) {
+        raceDay.text = "\(sessionDate.day ?? 0)"
+        raceMonth.text = "\(Constants.monthAbbreviations[(sessionDate.month ?? 1)-1])"
+        session.text = sessionName
+        time.text = sessionTime
+    }
+
+    static func nib() -> UINib {
+        return UINib(nibName: Identifiers.raceScheduleIndentifier, bundle: nil)
```

Comment:
all of these static func nib() can remove the return. i'm sure it will reduce a few warnings as well